less/pages/blog.less

```diff
@@ -48,4 +48,9 @@ article {
   h2, h3, h4, h5, h6 {
     padding-top: 2rem;
   };
+
+  pre[data-lang="console"] + pre:not([data-lang]) {
```

Comment:
As a possible next step, this could also target all non-console languages, as one can for example show expression-value pairs in Nix that way.

Reply:
As defined here, the second language has no language, and that should be good enough for most purposes.